mmocr/utils/ocr.py

```diff
@@ -364,7 +382,9 @@ def __init__(self,
             self.detect_model = revert_sync_batchnorm(self.detect_model)
 
         self.recog_model = None
-        if self.tr:
+        if self.tr and self.tr == 'Tesseract':
+            self.recog_model = self.tr
```

Comment:
Need an import check on `tesserocr` as well

Reply:
Roger, I will check all bugs tonight and try fix them in way